src/main/webapp/WEB-INF/web.xml

```diff
@@ -249,6 +254,15 @@
         <url-pattern>/*</url-pattern>
     </filter-mapping>
 
+    <filter-mapping>
+        <filter-name>crafter.httpHeadersRewriteFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+        <dispatcher>REQUEST</dispatcher>
```

Comment:
I think this shoud only run on `REQUEST`. Don't think it's worth it in terms of performance and it might be confusing if headers are set on forward or error when the original request that came is different.

Reply:
Updated to only run on `REQUEST`. Thanks.